src/package.json

```diff
@@ -12,7 +12,7 @@
     "start": "node app.js",
     "dev": "nodemon app.js",
     "build": "node esbuild & npm run copy:root & npm run copy:webp & npm run copy:png & npm run build:svg",
-    "build:svg": "svgo -f ./client/images/ -o ../dist/images",
+    "build:svg": "svgo -f ./client/images/ -r -o ../dist/images",
```

Comment:
Pet peeve of mine, but could we use non-abbreviated parameters here, to make it easier to understand for contributors? (I assume this would be `--recursive`?)

Reply:
I don't actually even know if `--recursive` would work.  `-f` was supposed to do the trick until I read https://github.com/svg/svgo/issues/1269

I haven't had time to read the entire readme, but as far as I know `-r` is a secret flag.  I haven't looked into the CLI's own help docs – maybe more current info there

Reply:
Secret flags? Well that's awesome :P

Reply:
Secret? It's in `--help`:
```sh
npx svgo --version # 3.0.2
npx svgo --help
Usage: svgo [options] [INPUT...]

Nodejs-based tool for optimizing SVG vector graphics files

Arguments:
  INPUT                      Alias to --input

Options:
  -v, --version              output the version number
  -i, --input <INPUT...>     Input files, "-" for STDIN
  -s, --string <STRING>      Input SVG data string
  -f, --folder <FOLDER>      Input folder, optimize and rewrite all *.svg files
  -o, --output <OUTPUT...>   Output file or folder (by default the same as the input), "-" for STDOUT
  -p, --precision <INTEGER>  Set number of digits in the fractional part, overrides plugins params
  --config <CONFIG>          Custom config file, only .js is supported
  --datauri <FORMAT>         Output as Data URI string (base64), URI encoded (enc) or unencoded (unenc)
  --multipass                Pass over SVGs multiple times to ensure all optimizations are applied
  --pretty                   Make SVG pretty printed
  --indent <INTEGER>         Indent number when pretty printing SVGs
  --eol <EOL>                Line break to use when outputting SVG: lf, crlf. If unspecified, uses platform default.
  --final-newline            Ensure SVG ends with a line break
  -r, --recursive            Use with '--folder'. Optimizes *.svg files in folders recursively.
  --exclude <PATTERN...>     Use with '--folder'. Exclude files matching regular expression pattern.
  -q, --quiet                Only output error messages, not regular status messages
  --show-plugins             Show available plugins and exit
  --no-color                 Output plain text without color
  -h, --help                 display help for comman
```